test/common/util.js

```diff
@@ -119,6 +124,12 @@ function handleBrowserConsole(consoleMessage) {
     }
 }
 
+function _onRequestFailed(request) {
+    if (request.failure()) {
+        console.error(`${request.url()} --> ${request.failure().errorText}`);
```

Comment:
I don't have any idea about the API, but, is it ok to call `request.failure()` twice?

Reply:
I think so https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#requestfailure

Reply:
first time check if null, then get the text

Reply:
:+1: